### DISCUSSION TITLE
Supply Path not showing

### DISCUSSION THREAD

**Discussion by zipus** [frePPLe/frepple#564] [Q&A] [answered]
I'm trying to translate the logic from our ERP (Tryton) to Frepple.
After reading through the docs and with the steps wizard I have tried to create the following on Frepple, also I've check the specific section of common errors.
I'm sure that our logic is much simpler than Frepple can handle. I will base my example as if we have to produce one personalized table.

To ease out the example, let's assume that all the things are done in the same location as the stock.
In my ERP I have the following products with inventories:
- Personalized table XYZ - 0 units
- Legs - 4 units
- Top wood - 1 unit

I have a bom list like:
- Inputs:
  > 4 x legs
  > 1 x top wood
- Outputs
  > 1 x Personalized table XYZ

And the routing steps are without any intermediate products, there are just a list with times and sequence / priority:
- Check integrity (need employee time: 5 min per unit)
- Make Holes (need employee time: 5 min per unit, resource drill used also during 5 min)
- Union (need employee time: 5 min per unit)
- Stamp custom name or designs
- Varnish (need employee time: 5 min per unit, resource painting room available during 5 min)

We don't have, and if it's possible we don't prefer to have intermediate products, that I'm sure it's the best aproach to consume in the union and also produce on the union.

Did not have any problems creating the locations, demand, inventory, customers, employees.
Items and inventory for the 3 products worked well. XYZ as make to order and inputs as make to stock.
For operations, I created:
- name: op_XYZ, type: routing, item: None
- name: op_XYZ_check_integrity, type: 'time_per', duration: 5 min, owner: op_XYZ
- name: op_XYZ_make_holes, type: 'time_per', duration: 5 min, owner: op_XYZ
- name: op_XYZ_union, type: 'time_per', duration: 5 min, owner: op_XYZ
- name: op_XYZ_stamp, type: 'time_per', duration: 5 min, owner: op_XYZ
- name: op_XYZ_varnish, type: 'time_per', duration: 5 min, owner: op_XYZ

For operationmaterial:
- operation: XYZ, item: leg, quantity: -4, type: start
- operation: XYZ, item: top wood, quantity: -1, type: start
- operation: XYZ, item: Personalized table XYZ, quantity: 1, type: end

Also I have created the resources and operationresource relating the op_XYZ with time in each resource but I don't think are relevant by now, because the problem I found is not having any supply path on the item Personalized table XYZ.
But I don't understand why because following the docs, there is a operation material of type end!
What I'm missing to see my supply path?
Thanks in advance for any help.

**Comment by jdetaeye** [comment, marked as answer]
>For operationmaterial:
>operation: XYZ, item: leg, quantity: -4, type: start
>operation: XYZ, item: top wood, quantity: -1, type: start
>operation: XYZ, item: Personalized table XYZ, quantity: 1, type: end

If these are on the top routing operation, try moving them to the child step operation of the routing.

**Comment by zipus** [comment]
Following the proposal, I moved the materials to `op_XYZ_union` and leave other operations `op_XYZ_*` with no material.
The operation, `op_XYZ_varnish`, is 5 min per unit. (others also).
If I produce 1 unit of `Personalized table XYZ`, it generates a `manufacturingorder` proposal for 5 min. All is okey.
But if I produce 10 units of `Personalized table XYZ`, it generates a `manufacturingorder` proposal for 5 min, which is not correct, as it should be 5min*10units as it's a time_per operation.
What I'm missing?

**Comment by zipus** [comment]
I don't know as I can not see any guideline about how Frepple community expects to use the discussions, I mention you @jdetaeye to know if I should create a new discussion since now the problem is not about the same subject as the topic?

**Comment by zipus** [comment]
>     * name: op_XYZ_check_integrity, type: 'time_per', duration: 5 min, owner: op_XYZ
>     * name: op_XYZ_make_holes, type: 'time_per', duration: 5 min, owner: op_XYZ
>     * name: op_XYZ_union, type: 'time_per', duration: 5 min, owner: op_XYZ
>     * name: op_XYZ_stamp, type: 'time_per', duration: 5 min, owner: op_XYZ
>     * name: op_XYZ_varnish, type: 'time_per', duration: 5 min, owner: op_XYZ

> But if I produce 10 units of Personalized table XYZ, it generates a manufacturingorder proposal for 5 min, which is not correct, as it should be 5min*10units as it's a time_per operation.
> What I'm missing?

So I auto-reply me, I just need to use `duration_per` instead of `duration`.
